lib/browser/init.js

```diff
@@ -162,8 +162,8 @@ require('./api/protocol')
 // Set main startup script of the app.
 const mainStartupScript = packageJson.main || 'index.js'
 
-// Workaround for electron/electron#5050
-if (process.platform === 'linux' && process.env.XDG_CURRENT_DESKTOP === 'Pantheon') {
+// Workaround for electron/electron#5050 and electron/electron#9046
+if (process.platform === 'linux' && ['Pantheon', 'Unity:Unity7'].includes(process.env.XDG_CURRENT_DESKTOP)) {
```

Comment:
just fyi - this is fixed in chromium 60 https://chromium.googlesource.com/chromium/src/base/+/dafe55da01f168f58f25a9d0e93e571bed0b47a8%5E%21/